cpp/src/arrow/adapters/orc/adapter_test.cc

```diff
@@ -15,20 +15,35 @@
 // specific language governing permissions and limitations
 // under the License.
 
-#include <string>
-
 #include "arrow/adapters/orc/adapter.h"
-#include "arrow/array.h"
-#include "arrow/io/api.h"
 
 #include <gtest/gtest.h>
+
 #include <orc/OrcFile.hh>
+#include <string>
+
+#include "arrow/adapters/orc/adapter_util.h"
+#include "arrow/api.h"
```

Comment:
`api.h` files are much too heavyweight, please make the inclusions more fine-grained.

Reply:
Ah I really didn't see this one..Fixed.:)